benchmarks/project/src/main/resources/application.conf

```diff
@@ -1,5 +1,7 @@
 app {
   brokerUrl = "localhost:26500"
+  brokerUrl = ${?ZEEBE_ADDRESS}
```

Comment:
How does this work with the line above? Is this only evaluated and assigned  when it exists?

Reply:
Yes that's I guess the meaning of the `?`, so if `ZEEBE_ADDRESS` does not exists it's ignored